test/flux_calculator_tests.jl

```diff
@@ -50,3 +63,222 @@ end
+        )
+        Y_init = (; ρ = ones(boundary_space) .* 1.2, T = ones(boundary_space) .* 310, q = zeros(boundary_space))
+        integrator = (; Y_init..., p = p)
+        atmos_sim = TestAtmos(params, Y_init, nothing, integrator)
+
+        # ocean
+        p = (;
+            F_aero = zeros(boundary_space),
+            z0m = FT(0.01),
+            z0b = FT(0.01),
+            beta = ones(boundary_space),
+            α = ones(boundary_space) .* 0.5,
+            q = zeros(boundary_space),
+            Cd = FT(0.01),
+            Ch = FT(0.01),
+            area_fraction = ones(boundary_space) .* 0.5,
+        )
+        Y_init = (; T = ones(boundary_space) .* 300.0)
+        integrator = (; Y_init..., p = p)
+        ocean_sim = TestOcean(nothing, Y_init, nothing, integrator)
+
+        # ocean
+        ocean_sim2 = TestOcean(nothing, Y_init, nothing, integrator)
+
+        model_sims = (; atmos_sim, ocean_sim, ocean_sim2)
+
+        coupler_cache_names = (
+            :T_S,
+            :albedo,
+            :F_R_sfc,
+            :F_R_toa,
+            :P_liq,
+            :P_snow,
+            :P_net,
+            :F_turb_energy,
+            :F_turb_ρτxz,
+            :F_turb_ρτyz,
+            :F_turb_moisture,
+        )
+        fields = NamedTuple{coupler_cache_names}(ntuple(i -> Fields.zeros(boundary_space), length(coupler_cache_names)))
+
+        # calculate turbulent fluxes
+        thermo_params = get_thermo_params(atmos_sim)
+        partitioned_turbulent_fluxes!(model_sims, fields, boundary_space, scheme, thermo_params)
+
+        # calculating the fluxes twice ensures that no accumulation occurred
+        partitioned_turbulent_fluxes!(model_sims, fields, boundary_space, scheme, thermo_params)
+
+        windspeed = @. hypot(atmos_sim.integrator.p.u, atmos_sim.integrator.p.v)
+
+        thermo_params = get_thermo_params(atmos_sim)
+        thermo_state_int = Interfacer.get_field(atmos_sim, Val(:thermo_state_int))
+
+        surface_thermo_states = similar(thermo_state_int)
+        Fields.bycolumn(boundary_space) do colidx
+            surface_thermo_states[colidx] .=
+                surface_thermo_state(ocean_sim, thermo_params, thermo_state_int[colidx], colidx)
+        end
+
+        if scheme == BulkScheme()
+            ρ_sfc = Interfacer.get_field(atmos_sim, Val(:air_density))
+            cpm = TD.cv_m.(thermo_params, thermo_state_int) .+ TD.gas_constant_air.(thermo_params, thermo_state_int) # cp = R + cv
+            gz =
+                (
+                    Interfacer.get_field(atmos_sim, Val(:height_int)) .-
+                    Interfacer.get_field(atmos_sim, Val(:height_sfc))
+                ) .* 9.81
+            shf_analytical = @. (cpm * (ocean_sim.integrator.T - atmos_sim.integrator.T) - gz) *
+               ocean_sim.integrator.p.Ch *
+               ρ_sfc *
+               windspeed #-ρ_sfc * Ch * windspeed(sc) * (cp_m * ΔT + ΔΦ)
+
+            colidx = Fields.ColumnIndex{2}((1, 1), 73)
```

Comment:
why 73 here?

Reply:
It's random - copied over from another test

Reply:
Sure!